indigo_api/templatetags/indigo.py

```diff
@@ -4,6 +4,203 @@
 register = template.Library()
 
 
+class Beautifier:
```

Comment:
I think a better home for this would be `toc/base.py`. Also maybe `CommencementsBeautifier` is a less generic name?